source/Hardpoint.h

```diff
@@ -19,6 +19,7 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include "Angle.h"
 #include "Point.h"
 
+#include <utility>
```

Comment:
This isn't utilized anywhere here, is it?
```suggestion
```

Reply:
Maybe. We'll see if anything breaks.